tck/src/main/java/org/eclipse/microprofile/fault/tolerance/tck/RetryConditionTest.java

```diff
@@ -62,6 +63,9 @@
  */
 public class RetryConditionTest extends Arquillian {
 
+    private final static String SIMULATED_EXCEPTION_MESSAGE = "Simulated error";
+    private final static String SIMULATED_RUNTIME_EXCEPTION_MESSAGE = "Test Exception - Simulated error";
```

Comment:
Changing these strings to constants is fine, but these strings are also used in `AsyncRetryClient`.

Please either leave the strings in the tests, or update `AsyncRetryClient` to use the same constants. Using the constants in one place but not the other gives you none of the benefits of using a constant and makes it harder to see that both parts of the test are using the same string.